STREAMING/AddModelToCreatorBudget.md

```diff
@@ -8,6 +8,16 @@ aliases: ["0x0BC3144DEB678666"]
 // 0x0BC3144DEB678666 0xC0E83320
 BOOL ADD_MODEL_TO_CREATOR_BUDGET(Hash modelHash);
 ```
+Add model memory size.
+In GTA:O call **Model Memory**
+https://youtu.be/AqQrs-4fWbw?t=222
```

Comment:
Something like "Adds the models required memory to the creator budget" also linking to a YouTube video is probably frowned upon for documentation as it has the possibility of being deleted later (somewhat similar to pastebins)